### PR TITLE
doc: add UserSession global and python documentation

### PR DESCRIPTION
**Pull request by PawelMorawian**

### CODE REVIEW

File: concepts/usersession.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: User Session
+---
+
+A user session can be created to track interactions with you LLM app.
+Those interactions are visible in the analytics tab of the platform.
```
Comment (willydouhard) [inline concepts/usersession.mdx:6]:
```suggestion
User sessions are then used to compute metrics on the dashboard.
```

File: concepts/usersession.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+You can simply create a session with:
+
+<CodeGroup>
+
+```python
+from chainlit_client import ChainlitClient
+
+client = ChainlitClient()
+
+await client.api.create_user_session(is_interactive=True, anon_participant_identifier="Alex")
```
Comment (willydouhard) [inline concepts/usersession.mdx:21]: I would give two examples
1. with an authenticated user
2. with anon_participant identifier
and explain the difference

File: introduction.mdx [hunk 3]
```diff
@@ -21,5 +24,5 @@ Chainlit covers a wide range of use cases, from conversational applications to t
 3. **Product & User Analytics**: Gain insights into your users and how they use your product. Chainlit provides traditional analytics as well as LLM powered analytics like Topic Analysis.
 
 <Frame caption="Chainlit Dashboard">
-  <img src="/images/analytics.png" />
-</Frame>
\ No newline at end of file
+  <img src="/images/dashboard-overview.png" />
```
Comment (willydouhard) [inline introduction.mdx:27]: this image is outdated

File: python-client/api-reference/usersession.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+title: UserSession
+---
+
+Checkout the documentation about [UserSessions](/concepts/usersession) for more information
+
+## Creating a UserSession
+
+```python
+session = await client.api.create_user_session(
+    participant_identifier="<PARTICIPANT_UUID>",
```
Comment (willydouhard) [inline python-client/api-reference/usersession.mdx:11]: this is not an uuid this is any id from the monitored app system

File: python-client/api-reference/usersession.mdx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+<ParamField path="started_at" type="string">
+  Start datetime
+</ParamField>
+
+<ParamField path="ended_at" type="string">
+  End of session datetime
+</ParamField>
+
+<ParamField path="participant_identifier" type="uuid">
+  Identifier of the participant
```
Comment (willydouhard) [inline python-client/api-reference/usersession.mdx:30]: need more details

File: python-client/api-reference/usersession.mdx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+</ParamField>
+
+<ParamField path="anon_participant_identifier" type="string">
+  Anonymous user
```
Comment (willydouhard) [inline python-client/api-reference/usersession.mdx:34]: need more details

File: python-client/api-reference/usersession.mdx [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+</ParamField>
+
+<ParamField path="metadata" type="dict">
+  Metadata
```
Comment (willydouhard) [inline python-client/api-reference/usersession.mdx:38]: need more details (what is this useful for?)

**Review by willydouhard — CHANGES_REQUESTED** [review]

File: python-client/api-reference/usersession.mdx (hunk 7 above)
Comment (PawelMorawian) [reply python-client/api-reference/usersession.mdx:38]: I don't really know what we can add here.

File: python-client/api-reference/usersession.mdx (hunk 4 above)
Comment (PawelMorawian) [reply python-client/api-reference/usersession.mdx:11]: I don't understand what you mean. It is not a foreign key ?

**Review by willydouhard — APPROVED** [review]

**Pull request merged**